src/creature.cpp

```diff
@@ -260,7 +260,7 @@ void Creature::startAutoWalk()
 
 void Creature::startAutoWalk(Direction direction)
 {
-	Player* player = getPlayer();
+	auto player = getPlayer();
```

Comment:
Could this be `const auto`
```suggestion
	const auto player = getPlayer();
```